introduction/A_overview.md

```diff
@@ -1,8 +1,8 @@
 Phoenix is a web development framework written in Elixir which implements the server-side MVC pattern. Many of its components and concepts will seem familiar to those of us with experience in other web frameworks like Ruby on Rails or Python's Django.
 
-Phoenix provides the best of both worlds, high developer productivity _and_ high application performance. It also has some interesting new twists like channels for implementing realtime features and pre-compiled templates for blazing speed.
+Phoenix provides the best of both worlds: high developer productivity _and_ high application performance. It also has some interesting new twists like channels for implementing realtime features and pre-compiled templates for blazing speed.
 
-If you are already familiar with Elixir, great! If not, there are a number of places to learn. The [Elixir guides](http://elixir-lang.org/getting_started/1.html) are a great place to start. We also have a list of helpful resources in the [Learning Elixir and Erlang Guide](http://www.phoenixframework.org/docs/learning-elixir-and-erlang).
+If you are already familiar with Elixir, great! If not, there are a number of places to learn from. The [Elixir guides](http://elixir-lang.org/getting_started/1.html) are a great place to start. We also have a list of helpful resources in the [Learning Elixir and Erlang Guide](http://www.phoenixframework.org/docs/learning-elixir-and-erlang).
```

Comment:
Thinking out loud about this one. In this sentence, "it" is the implied object of the verb "learn". (If not, there are a number of places to learn it.) Explicitly adding the "it" is a little awkward. The preposition "from" doesn't play the same role in the sentence as the object.

For now, let's please leave the original as it is.